en/pyvespa/create-text-app.md

```diff
@@ -0,0 +1,62 @@
+---
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Create a text search application"
+---
+> Get started with the Python API to create and modify Vespa applications
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/vespa-engine/pyvespa/blob/master/docs/sphinx/source/create-text-app.ipynb)
+
+This self-contained tutorial will create a basic text search application from scratch based on the MS MARCO dataset, similar to Vespa's [text search tutorials](https://docs.vespa.ai/documentation/tutorials/text-search.html). Visit [pyvespa reference API](https://pyvespa.readthedocs.io/en/latest/reference-api.html) for more detailed information about the API presented here.
```

Comment:
Updated to a relative link as it is in the same repo. Tip: linebreaks makes later reviews easier, text is like code :-)
```suggestion
This self-contained tutorial will create a basic text search application from scratch based on the MS MARCO dataset,
similar to Vespa's [text search tutorials](../tutorials/text-search.html).
Visit [pyvespa reference API](https://pyvespa.readthedocs.io/en/latest/reference-api.html)
for more detailed information about the API presented here.
```